### ISSUE TITLE
template is undefined

### ISSUE THREAD

**Issue by anthony-driverloan** [niftylettuce/node-email-templates#131]
Help required with this. Am i doing something wrong?

var path = require('path');
var templatesDir  = path.join(__dirname, 'templates');
var emailTemplates = require('email-templates');

  emailTemplates(templatesDir, function(err, template) {

    // Render a single email with one template
    var locals = { fName: 'user firstname' };
    // Render a single email with one template

    template('confirm-email', locals, function(err, html, text) {

    });

  });

i keep getting this error.

template('confirm-email', locals, function(err, html, text) {
  ^
TypeError: undefined is not a function

**Comment by anthony-driverloan** [comment]
Sorry found out what i was doing wrong

**Comment by Deminetix** [comment]
Thanks for sharing....

**Comment by reedyrm** [comment]
I found this after running into a similiar problem.  I was able to track down the problem of one of my handlebar helpers that I had registered.  It was throwing an exception which was email-templates to throw an error saying that it could not find "<my directory structure>/undefined".  I slowly removed all the custom helpers in the template until it worked.

Hopefully if anyone finds this message, it helps them with this problem
